pkg/process/monitor/process_monitor.go

```diff
@@ -35,6 +36,42 @@ var (
 	}
 )
 
+// processMonitorTelemertry
+type processMonitorTelemertry struct {
+	// process monitor will process :
+	//  o events refer to netlink events received (not only exec and exit)
+	//  o exec process netlink events
+	//  o exit process netlink events
+	//  o restart the netlink connection
+	//
+	//  o reinit_failed is the number of failed re-initialisation after a netlink restart due to an error
+	//  o process_scan_failed would be > 0 if initial process scan failed
+	//  o callback_called numbers of callback called
+	events  *telemetry.Counter
+	exec    *telemetry.Counter
+	exit    *telemetry.Counter
+	restart *telemetry.Counter
+
+	reinitFailed      *telemetry.Counter
+	processScanFailed *telemetry.Counter
+	callbackExecuted  *telemetry.Counter
+}
+
+func (pmt *processMonitorTelemertry) initialize() {
```

Comment:
I feel this is an anti-pattern in Go. why not having a constructor function that returns a `processMonitorTelemetry ` instance?

Reply:
I moved to newProcessMonitorTelemetry()